src/CabanaPD_Boundary.hpp

```diff
@@ -158,9 +151,16 @@ struct BoundaryIndexSpace<MemorySpace, RegionBoundary<GeometryType>>
         _timer.stop();
     }
 
-    template <class ExecSpace, class Particles>
-    void update( ExecSpace, Particles particles,
-                 RegionBoundary<GeometryType> region )
+    // Iterate over all regions.
+    template <class ExecSpace, class Particles, class... RegionType>
+    void update( ExecSpace space, Particles particles, RegionType... region )
+    {
+        ( [&] { update( space, particles, region ); }(), ... );
```

Comment:
Is a lambda necessary for the fold here?

Reply:
Removed